src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -159,7 +172,8 @@ class AsyncDropDown extends React.Component {
         onStateChange={this.handleStateChange}
         onChange={this.handleChange}
         initialIsOpen={startOpen}
-        selectedItem={this.state.selectedItem}>
+        selectedItem={this.state.selectedItem}
+        isOpen={this.state.page && this.state.inputValue}>
```

Comment:
Hadde et problem med at nedtrekkslista lukket seg når man prøvde å bytte sidetall, så måtte på en måte signalisere at den må holde seg åpen. 

Default oppførsel om at søket åpner seg når man bare trykkeri input-feltet mister man nå hvis man bruker komponenten med sidetall. Den oppførselen dukker opp når man trykker en tast, for da dukker resultatet opp med input="" mens søket loader med den nye inputen. Det kan man forsåvidt se i loading-ikonet, og varer ikke så fryktelig lenge, men likevel litt dumt kanskje?

Vanskelig å forklare, men tok et opptak av det også: https://user-images.githubusercontent.com/55005434/103516340-a5ce5a80-4e70-11eb-851a-52de3d03b1cd.mov

Reply:
Kva skjer om du trykker pil ned i feltet? Dukker tomt søketreff opp då? Det kunne jo vore en mulighet.

Reply:
nå reagerer den bare på input, men kan absolutt legge til at den skal reagere på pil ned!

Reply:
Husket litt feil, for det var ikke ved trykk at dropdownmenyen åpnet seg, men andre steder holder den seg åpen når man fjerner den inputen man har skrevet inn. Skal jeg prøve å heller gjøre sånn at hvis man gir noe input, eller pil ned, så åpnes dropdownmenyen og så lukkes den ikke igjen før man trykker utenfor?

Reply:
@gunnarvelle Skal jeg legge til at den åpner tomt søk ved pil ned for all bruk av AsyncDropdown, eller bare for relaterte artikler?

Reply:
Kan gjøre det som standard tenker eg.